app/components/reporting/reporting-line-items-list/reporting-line-items-list.js

```diff
@@ -8,10 +8,26 @@ angular.module('lineItems.list', [
   .config(function ($routeProvider, routes) {
     $routeProvider
       .when('/cms/app/line-items/', {
-        controller: function($scope, $window, LineItem) {
+        controller: function($modal, $scope, $window, LineItem) {
           $window.document.title = routes.CMS_NAMESPACE + ' | Line Items';
 
           $scope.modelFactory = LineItem;
+
+          $scope.lineItemExportModal = function () {
+            return $modal.open({
+              templateUrl: routes.PARTIALS_URL + 'modals/line-item-export-modal.html',
+              controller: 'LineitemexportmodalCtrl',
```

Comment:
To keep in line with more recent code style rename this to `LineItemExportModal`.